variables.tf

```diff
@@ -55,6 +55,12 @@ variable "cos_instance_name" {
   default     = null
 }
 
+variable "existing_cos_instance_id" {
+  description = "ID of an existing cos instance where the bucket will be created"
```

Comment:
Lets be consistent..

The GUID of an existing COS instance, required if 'var.create_cos_instance' is false.

Reply:
Also update the description for var.cos_instance_name to be consistent too...

The name to give the COS instance that will be provisioned by this module. This variable will be ignored if a value is passed for 'var.existing_cos_instance_id'.